From cc05052d379e9a47c8622c3d1bbb99a31bfba713 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal.vasilek@nic.cz>
Date: Tue, 17 Jan 2023 11:53:12 +0100
Subject: [PATCH] syncthing: do not prevent service from running

Enable syncthing in /etc/config by default to make sure when we
enable the service, syncthing can run without having to manually
enable it in the config.

Users, by default, expect that the service will run automatically and that
they will configure it while the service is running to see what was changed
and what is the behavior of each function there. This change is going to enable
syncthing to run after installing it instead of tweaking the config manually
(in LuCI or by hand). This is somehow different than how OpenWrt does it. They
force users to configure the app before running. Once they are confident that
the configuration looks good, they enable it, which is also good behavior.
Still, we need to improve the user experience and somehow make it different as
we want that even non-IT guys/girls can configure what they need and install
what they need in our simplified interface called (re)Foris, where we provided
list of packages, which they can install and if it does not run by default, it
looks like something is wrong. For syncthing, we also provide
Transmission WebApp, the integration for Turris OS, which you can find on the
landing page and easily click on it and start configuring it.
---
 utils/syncthing/files/syncthing.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/syncthing/files/syncthing.conf b/utils/syncthing/files/syncthing.conf
index 193f5e9ea..17f0fc998 100644
--- a/utils/syncthing/files/syncthing.conf
+++ b/utils/syncthing/files/syncthing.conf
@@ -1,6 +1,6 @@

 config syncthing 'syncthing'
-	option enabled '0'
+	option enabled '1'

 	option gui_address 'http://0.0.0.0:8384'

--
2.34.1

